src/modules/fw_autotune_attitude_control/fw_autotune_attitude_control_params.c

```diff
@@ -121,3 +121,53 @@ PARAM_DEFINE_INT32(FW_AT_AXES, 3);
  * @group Autotune
  */
 PARAM_DEFINE_INT32(FW_AT_MAN_AUX, 0);
+/**
```

Comment:
```suggestion

/**
```
newline missing above